var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -400,6 +402,9 @@ class QuantumEspresso(CMakePackage, Package):
     # extlibs_makefile updated to work with fujitsu compilers
     patch("fj-fox.patch", when="+patch %fj")
 
+    # gipaw.x will only be installed with cmake if the qe-gipaw version is >= 5c4a4ce.
+    patch("gipaw-eccee44.patch", when="@7.2+gipaw build_system=cmake")
```

Comment:
```suggestion
    patch("gipaw-eccee44.patch", when="@:7.2+gipaw build_system=cmake")
```
This patch should apply to all versions 7.2 or prior that support gipaw

Reply:
This patch can only be applied on QE 7.2 release.

Reply:
Then we need a

```
conflicts("+gipaw build_system=cmake", when="@:7.1")
```

Reply:
Is it OK if I take care of this in my staged PR https://github.com/spack/spack/pull/41225 so I have one fewer PR to shepherd.

Reply:
Added on line 265 above.